src/client/pages/ConsentsCommunication.tsx

```diff
@@ -49,6 +120,12 @@ export const ConsentsCommunicationPage = () => {
     (consent) => !consent.id.includes('_optout'),
   );
 
+  // @TODO: AB TEST: OneConsentTest.
```

Comment:
this todo still needed?

Reply:
Until the test is done and this code is deleted, yes!